slices/findd.go

```diff
@@ -0,0 +1,21 @@
+package slices
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var digitRegexp = regexp.MustCompile("[0-9]+")
+
+//FindFirstConsecutiveDigits returns an slice containg the first sequence of integers in the input string slice
+func FindFirstConsecutiveDigits(sequence []string) []int {
```

Comment:
This is a freestyle exercise, so a thoroughly code review is highly appreciated @JuanCalcagno416, thanks in advance!